samples/show-case/cloud-azure/sample.md

```diff
@@ -14,6 +14,23 @@ This sample implements a fictional store that can be deployed to Azure. It is di
 ## Feature usage
 
 
+### Azure Storage Queues Transport
+
+All endpoints in the solution communicate using the [Azure Storage Queues transport](/nservicebus/azure-storage-queues/index.md). The sample has been configured to use the Azure Storage Emulator. 
+
+
+### Azure Storage Persistence
+
+The endpoints in the solution persist data using the [Azure Storage persistence](/nservicebus/azure-storage-persistence/index.md). This persistence is used to store subscription, timeout, and saga data.
+
+NOTE: The Azure Storage persistence does not support collection types. The `ProcessOrderSaga` avoids this issue by combining a collection of `ProductIds` into a single string for persistence.
```

Comment:
Oh. It threw an exception when I first ported the sample but that was before that PR was merged. I will make sure I am running the latest persistence code and revert to the original Saga code to see if it now works.